docs/_includes/floating-label.html

```diff
@@ -1,431 +1,350 @@
+    textbox otherwise.</p>
 
-    <h3>Floating Label with no Value</h3>
-    <p>If the textbox has no value, the label can be positioned inside the textbox.</p>
+  <h3>Floating Label with no Value</h3>
+  <p>If the textbox has no value, the label can be positioned inside the textbox.</p>
 
-    <div class="demo">
-        <div class="demo__inner">
-            <span class="floating-label">
-                <label class="floating-label__label" for="first-name-1">First Name</label>
-                <span class="textbox">
-                    <input class="textbox__control" id="first-name-1" type="text" />
-                </span>
-            </span>
-        </div>
+  <div class="demo">
+    <div class="demo__inner">
+      <span class="floating-label">
+        <label class="floating-label__label" for="first-name-1">First Name</label>
+        <span class="textbox">
+          <input class="textbox__control" id="first-name-1" type="text" />
+        </span>
+      </span>
     </div>
+  </div>
 
-    {% highlight html %}
-<span class="floating-label">
+  {% highlight html %}
+  <span class="floating-label">
     <label class="floating-label__label" for="first-name">First Name</label>
     <span class="textbox">
-        <input class="textbox__control" id="first-name" type="text" />
+      <input class="textbox__control" id="first-name" type="text" />
     </span>
-</span>
-    {% endhighlight %}
-
-    <h3>Floating Label with Value</h3>
-    <p>If the textbox has a value, the label must be positioned above the textbox.</p>
-
-    <div class="demo">
-        <div class="demo__inner">
-            <span class="floating-label">
-                <label class="floating-label__label" for="last-name-1">Last Name</label>
-                <span class="textbox">
-                    <input class="textbox__control" id="last-name-1" type="text" value="Smith" />
-                </span>
-            </span>
-        </div>
+  </span>
+  {% endhighlight %}
+
+  <h3>Floating Label with Value</h3>
+  <p>If the textbox has a value, the label must be positioned above the textbox.</p>
+
+  <div class="demo">
+    <div class="demo__inner">
+      <span class="floating-label">
+        <label class="floating-label__label" for="last-name-1">Last Name</label>
+        <span class="textbox">
+          <input class="textbox__control" id="last-name-1" type="text" value="Smith" />
+        </span>
+      </span>
     </div>
+  </div>
 
-    {% highlight html %}
-<span class="floating-label">
+  {% highlight html %}
+  <span class="floating-label">
     <label class="floating-label__label" for="last-name">Last Name</label>
     <span class="textbox">
-        <input class="textbox__control" id="last-name" type="text" value="Smith" />
+      <input class="textbox__control" id="last-name" type="text" value="Smith" />
     </span>
-</span>
-    {% endhighlight %}
+  </span>
+  {% endhighlight %}
 
-    <h3>Large floating label</h3>
+  <h3>Large floating label</h3>
```

Comment:
We typically add a short paragraph indicating the modifier to be used and the impact of the modifier. Can we add that here?